src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
+    """
+    Logs dictionary data from leader
+    """
+    if data_type == 'leader':
+        leader_dict = {
+            'Time' : time.time(), 
+            'Input' : data._distance,
+            'Speed' : data._speed,
+        }
+        write_to_file(leader_dict, data_type)
+    elif data_type == 'follower':
+        follower_dict = {
+            'Time' : time.time(), 
+            'Distance' : data._distance,
+            'Speed' : data._speed,
+            'Turn Angle' : data._turn_angle,
+            'Yaw' : data._yaw,
+        }
+        write_to_file(follower_dict, data_type)
+    else:
+        pass
+
+def write_to_file(data, file_name):
```

Comment:
The file and its name should be auto generated and unique.

Reply:
add unique naming scheme